src/test_suite/test_suite.py

```
@@ -274,9 +274,12 @@ def create_fixtures(
     # Initialize shared library
     for target in shared_libraries:
         # Load in and initialize shared libraries
-        lib = ctypes.CDLL(target)
-        lib.sol_compat_init(log_level)
-        globals.target_libraries[target] = lib
+        try:
+            lib = ctypes.CDLL(target)
+            lib.sol_compat_init(log_level)
+            globals.target_libraries[target] = lib
+        except:
```

you should probably have it explicitly catch the `undefined symbol` error so that it doesnt mask other errors (e.g. missing file)

---

Sure, I just copied how it's already done. 


https://github.com/firedancer-io/solana-conformance/blob/86c505592dfdd4a7ae3b51d6691f710796158a6b/src/test_suite/test_suite.py#L103-L109

I don't see an exception in the docs at first glance that seems to fit this
https://docs.python.org/3/library/ctypes.html#loading-shared-libraries